www/main/addons/ceremonies/components/CeremonyPane.jsx

```diff
@@ -48,13 +69,43 @@ const CeremonyPane = ({ token, name, id, onScreenClose }) => {
       setIsCeremonyBani(true);
     }
     onScreenClose();
-    if (currentTheme !== theme.key) {
-      applyTheme(theme, null, setTheme, setThemeBg);
+    if (currentTheme !== parsedTheme.key) {
+      applyTheme(parsedTheme, null, setTheme, setThemeBg);
+    }
+    if (multipaneId !== null) {
+      switch (multipaneId) {
+        case 1:
+          setPane1({
+            ...pane1,
+            content: i18n.t('MULTI_PANE.SHABAD'),
+            baniType: 'ceremony',
+            activeShabad: currentCeremony,
+          });
+          break;
+        case 2:
+          setPane2({
+            ...pane2,
+            content: i18n.t('MULTI_PANE.SHABAD'),
+            baniType: 'ceremony',
+            activeShabad: currentCeremony,
+          });
+          break;
+        case 3:
+          setPane3({
+            ...pane3,
+            content: i18n.t('MULTI_PANE.SHABAD'),
+            baniType: 'ceremony',
+            activeShabad: currentCeremony,
+          });
+          break;
+        default:
```

Comment:
This switch statement is also being used at multiple places with different variables. We should have a common function that can return the value we need from these statements.